azure-deployment/README.md

```diff
@@ -0,0 +1,73 @@
+# Azure Deployment script
```

Comment:
Add a reference to this README file in the main README file (the one that's on the root). Under deployment section there should be a link to this folder.

In the end of this file add a comment to go back to the main README file (the on in the root).

Reply:
done